### PR TITLE
Added intro paragraphs to each major section of the docs

### PR DESCRIPTION
**Pull request by rtapella**
Issue:
https://github.com/MAAP-Project/maap-documentation/issues/168

### CODE REVIEW

File: docs/source/science_examples.rst [hunk 1]
```diff
@@ -1,6 +1,10 @@
 Science Examples
 =======================================
 
+Here you will find a variety of real-world examples that may help you create a custom Jupyter notebook for your research. These notebooks have data-set specific analysis and visualization examples, and will also provide examples of analysis, data access, tabulation and map-visualization.
```
Comment (grallewellyn) [inline docs/source/science_examples.rst:4]: data-set or dataset?

Comment (rtapella) [reply docs/source/science_examples.rst]: dataset is more consistent with the rest of the docs
changed

**Review by grallewellyn — APPROVED** [review]
Looks good to me

File: docs/source/getting_started/running_at_scale.ipynb
Comment (wildintellect) [inline docs/source/getting_started/running_at_scale.ipynb:1]: I'm not sure I understand the need for this header level change.

File: docs/source/release_notes.rst [hunk 2]
```diff
@@ -1,6 +1,8 @@
 Release Notes
 =======================================
 
+Release notes can be found here. This is typically more comprehensive than the announcements on the GitHub Discussion board: https://github.com/orgs/MAAP-Project/discussions/categories/announcements.
```
Comment (wildintellect) [inline docs/source/release_notes.rst:4]: > Release notes can be found here.
Maybe better to explain what's in a release note?
- Major system changes
- Fixes
- Feature Removal notices
etc...

File: docs/source/system_reference.rst [hunk 3]
```diff
@@ -1,6 +1,10 @@
 System Reference Guide
 =======================================
 
+This section of documentation is intended as a technical reference for the functionality of the MAAP platform. If you are interested in learning about a particular part of the MAAP platform, this section will contain the canonical reference. 
+
+It is often referenced from the application-focused tutorial notebooks.
```
Comment (wildintellect) [inline docs/source/system_reference.rst:6]:
```suggestion
It is often referenced from the application-focused Science and Technical tutorials.
```

**Review by wildintellect — COMMENTED** [review]
No particular blockers, but some suggestions to consider to change the phrasing and focus.

File: docs/source/getting_started/running_at_scale.ipynb
Comment (rtapella) [reply docs/source/getting_started/running_at_scale.ipynb:1]: I noticed that the conclusion didn't show up in the left-hand navigation tree b/c it was lower-level. I didn't think it deserved its own page. But it felt like it should look like an "end" section in the navigation list.

Do you buy that? 😆

File: docs/source/system_reference.rst (hunk 3 above)
Comment (rtapella) [reply docs/source/system_reference.rst]: updated

**Pull request merged**